.github/workflows/build.yml

```diff
@@ -30,5 +30,8 @@ jobs:
           npm run build
           script/check-generated-files
 
-      - name: Test corpus & parse examples
-        run: npm test
+      - name: Run parse tests
+        run: npm run test-parse
+
+      - name: Run stack graph tests
+        run: npm run test-stack-graphs
```

Comment:
I have made running the stack graph tests a separate CI job. This is mainly to make it obvious which test group is failing, and to make it easier to temporarily disable stack graph tests in CI, if this is blocking contributions. See the changes to README for more on that.